src/azure-cli/azure/cli/command_modules/batch/tests/latest/test_batch_mgmt_commands.py

```diff
@@ -341,65 +332,3 @@ def test_batch_application_cmd(self, resource_group, storage_account):
         self.cmd('batch application delete -g {rg} -n {acc} --application-name {app} --yes')
         self.cmd('batch application list -g {rg} -n {acc}').assert_with_checks(self.is_empty())
 
-
-class BatchMgmtByosScenarioTests(ScenarioTest):
```

Comment:
I think we should keep this working, as it's testing an important case (user subscription mode).

Reply:
added back